src/sharedInterfaces/queryResult.ts

```diff
@@ -28,12 +47,35 @@ export interface QueryResultWebviewState {
     value?: string;
     messages: IMessage[];
     tabStates?: QueryResultTabStates;
+    isExecutionPlan?: boolean;
```

Comment:
Can we encapsulate execution plan related fields into one field like `executionPlanStates: ExecutionPlanStates` and put these fields under `ExecutionPlanStates`?